### ISSUE TITLE
Simple timer is completely broken

### ISSUE THREAD

**Issue by Adam314159** [online-go/online-go.com#669]
### Summary

Games under simple timer can timeout way before the clock reaches zero

### Environment

 * Operating system: Win 8 home 
 * Browser: Moz FF 61,0,1

### Steps To Reproduce:

1. Start a live challenge with Simple timer (tested on 25s)
2. See that timer shows more time than was chosen
3. Play a few moves, the clock seems to behave wrong (adding time and stuff)
4. Timeout suddenly even though your game clock shows many seconds left

### Additional Information:
Looks like more users are experiencing this https://forums.online-go.com/t/timer-incorrect/17440/3

**Comment by koshogs** [comment]
I think it would be worth disabling 'simple time' until it can be fixed.  As noted in multiple forum threads, the same time setting can be achieved with a single byo-yomi period or fischer x + x up to max x.

**Comment by roy7** [comment]
I mentioned simple time being fixed for bots in chat, and someone asked if the timers themselves were fixed. They went and tried it. A "10 seconds per move" game kept showing 30 seconds on their clocks instead of 10 seconds. I saw 10 seconds fine as a viewer, maybe it's only wrong (sometimes?) for the player? Another person also tried a test game and his timers looked correct.

Consider this 10 second/move game https://online-go.com/game/19130349 vs this screenshot of the timers based on 30 seconds instead https://i.imgur.com/JbLipWn.png.
